src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -492,7 +493,7 @@ export default class AmalgamatingBusinesses extends Mixins(AmalgamationMixin, Co
       name: business.businessInfo.legalName,
       email: business.authInfo.contacts[0].email,
       legalType: business.businessInfo.legalType,
-      address: business.addresses.registeredOffice.mailingAddress,
+      addresses: business.addresses,
```

Comment:
Now saving the whole address structure, so it can be reused if we set this business as the holding/primary (without needing to fetch the addresses again). Let me know how you feel about this, as the cost is extra memory usage. I don't mind reverting this if you think the other way was better.

Reply:
OK with me 👍